spec/features/work_packages/custom_actions/custom_actions_spec.rb

```diff
@@ -499,4 +499,32 @@
     find('[data-field-name="estimatedTime"]').click
     expect(page).to have_selector("#wp-#{work_package.id}-inline-edit--field-estimatedTime[disabled]")
   end
+
+  context 'with baseline enabled' do
+    let(:wp_table) { Pages::WorkPackagesTable.new(project) }
+    let(:query) do
+      create(:query,
+             name: 'Timestamps Query',
+             project:,
+             user:,
+             timestamps: ["P-1d", "PT0S"])
+    end
+
+    before do
+      create(:custom_action,
+             actions: [CustomActions::Actions::AssignedTo.new(value: nil)],
+             name: 'Unassign')
+    end
+
+    it 'executes the custom action (Regression#49588)' do
+      login_as(user)
+      wp_table.visit_query(query)
+      wp_page = wp_table.open_full_screen_by_link(work_package)
+
+      wp_page.ensure_page_loaded
+
+      wp_page.expect_custom_action('Unassign')
```

Comment:
I've tried this example out without this expectation and I don't see why it may be needed for timing reasons as `wp_page.ensure_page_loaded` covers this. Is this expectation there for another reason?

Reply:
The expectation was there, just to ensure we have the button present on the page. However it is kind of redundant as you pointed out, so I removed it.